src/test/java/org/reaktivity/nukleus/http_cache/internal/streams/server/ProxyCacheIT.java

```diff
@@ -127,6 +127,28 @@ public void shouldRequestCacheMaxAgeZeroAnd304() throws Exception
 
     @Test
     @Specification({
+            "${route}/proxy/controller",
+            "${streams}/cache.get.request.with.no-store/accept/client",
+            "${streams}/cache.get.request.with.no-store/connect/server",
+    })
+    public void shouldCacheGetRequestWithNoStore() throws Exception
```

Comment:
The name of test is a bit confusing. Shouldn't it be 'shouldNotCacheGetRequestWithNoStore'